decidim-proposals/spec/types/proposal_type_spec.rb

```diff
@@ -29,6 +34,70 @@ module Proposals
         end
       end
 
+      describe "when proposal has costs" do
```

Comment:
Add examples for when the proposal answer/costs are published or not

Reply:
done in [e1adc69](https://github.com/decidim/decidim/pull/14644/commits/e1adc69ddcf8fbe0a256c8004d9ab222f5a59471)